feat(workspace): add new data source query workspace tags (#6990)
huaweicloud/terraform-provider-huaweicloud: docs/data-sources/workspace_tags.md

---
subcategory: "Workspace"
layout: "huaweicloud"
page_title: "HuaweiCloud: huaweicloud_workspace_tags"
description: |-

>   Use this data source to query the desktop tags under a specified region within HuaweiCloud.

Reviewer: ```suggestion
  Use this data source to query tag list under all desktops  within HuaweiCloud.
```
